mkdocs.yml

```diff
@@ -89,6 +89,8 @@ nav:
       - Common Event Format: integrations/cef.md
     - SEKOIA.IO:
       - Activity logs: integrations/sekoiaio_activity_logs.md
+    - External tools:
```

Comment:
Should we not consider The hive as an Application (instead of an External tools)?